.github/workflows/ci.yml

```diff
@@ -23,6 +23,7 @@ jobs:
           - "++2.13.3 testsJVM/test"
           - "++2.13.4 testsJVM/test"
           - "++2.13.4 testsJS/test"
+          - "++2.13.4 testkit/test"
```

Comment:
Can we move these tests to the testsJVM project?

Reply:
Moved!